src/Microsoft.IdentityModel.Tokens/CryptoProviderCacheOptions.cs

```diff
@@ -0,0 +1,53 @@
+﻿//------------------------------------------------------------------------------------------------
+// <copyright file="TokenValidator.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
MIT license?

Reply:
Whoops, fixed here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1605/commits/af3243229ca3f4885398299f0d733066d3b9be46.